app/code/core/Mage/Sales/Model/Entity/Order/Address/Collection.php

```diff
@@ -34,6 +34,12 @@
 
 class Mage_Sales_Model_Entity_Order_Address_Collection extends Mage_Eav_Model_Entity_Collection_Abstract
 {
+    /**
+     * Model event prefix
+     *
+     * @var string
+     */
+    protected $_eventPrefix = 'eav_sales_order_address_collection';
```

Comment:
I don't think the `eav_` makes a lot of sense, does it? I understand why you added, so you didn't need to use `entity` in the prefix. I think that using `entity` is more logical here. Unless you are basing the even prefix on an existing example in the core code that you can show me.